### PR TITLE
UI - Add Customer Data and BackEnd Integration

### PR DESCRIPTION
**Pull request by RafaelMeha**
Added BackEnd Integration to get customers data

### CODE REVIEW

**Review by diogo-s4ntos — APPROVED** [review]

**Review by Silva096 — CHANGES_REQUESTED** [review]
We just need CustomerModel
We don't need CustomerManagementResponseDto

File: src/app/services/realestate-api.service.ts [hunk 1]
```diff
@@ -15,4 +16,8 @@ export class RealestateApiService {
   getAllReferenceData() : Observable<ReferenceDataResponseDto> {
     return this.httpCLient.get<ReferenceDataResponseDto>(this.baseUrl + 'ReferenceData')
   }
+
+  getAllCustomerData() : Observable<Array<CustomerModel>>{
```
Comment (Silva096) [inline src/app/services/realestate-api.service.ts:20]: We should create a new customer-api.service.ts to store it's endpoints
This one is for real estate

Comment (RafaelMeha) [reply src/app/services/realestate-api.service.ts]: Done

File: src/app/services/customer-api.service.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+@Injectable({
+  providedIn: 'root'
+})
+export class CustomerApiService {
+
+  baseUrl = "http://localhost:5152/";
+
+  constructor(private httpCLient: HttpClient) {}
+
+  getAllCustomerData() : Observable<Array<CustomerModel>>{
```
Comment (Silva096) [inline src/app/services/customer-api.service.ts:15]: Usually we use "CustomerModel[]" instead of <Array<CustomerModel>>

Comment (RafaelMeha) [reply src/app/services/customer-api.service.ts:15]: Done!

**Review by jonathan-osorio — APPROVED** [review]

**Review by Silva096 — APPROVED** [review]

**Pull request merged**